plugins/se.infomaker.contentrelations/README.md

```diff
@@ -1,23 +1,110 @@
 # Content relations plugin
-This plugin uses `Concept Backend` to search Writer OC backend for articles and images. From the search result, the
-user can drag and drop the article/image onto the article.
+
+This plugin uses a `Editorial Ba-proxy` to search Editorial OC backend for articles. From the search result, the
+user can drag and drop the related article onto the open article.
+
+## Dependency
+
+- This plugin requires `writer > 3.12.0` and depends on the `writer.UIPagination` class.
+- This plugin requires a contentHost (editorial ba-proxy) which is used for Open Content search requests towards the editorial Open Content, and needs configuration under contentHost (see below)
 
 ## Plugin configuration
-No `data` configuration needed. Note that the configuration for how to search and response fields are
-handled by `Concept Backend`.
+
+- `"tab": "main"` in this field, above the data section, you can specify in which tab the plugin should run, if this field is left out, it will render in a new tab
+- `"npDropLinkMatcher":` On drop, If a match is found params is updated with `isNpDroplink=true`
+- `"contentHost": {... ` BA-proxy config
+- `"contenttype": "Article",` Article contentType in OC
+- `"locale": "sv",` Locale used to print dates
+- `"defaultQueries": [ ` populate the query-dropdown with queries
+- `"icons": [... ` icons to use in results list, article.product (specified in propertyMap) will look for `icon.name`
 
 ```json
 {
-  "id": "se.infomaker.contentrelations",
-  "name": "contentrelations",
-  "url": "https://plugins.writer.infomaker.io/releases/{PLUGIN_VERSION}im-contentrelations.js",
-  "style": "https://plugins.writer.infomaker.io/releases/{PLUGIN_VERSION}im-contentrelations.css",
-  "enabled": true,
-  "mandatory": false
+    "id": "se.infomaker.contentrelations",
+    "name": "contentrelations",
+    "url": "http://localhost:5001/im-contentrelations.js",
+    "style": "http://localhost:5001/im-contentrelations.css",
+    "enabled": true,
+    "mandatory": false,
+    "tab": "main",
```

Comment:
maybe remove this, since it will override the normal tab placement (`"ContentRelations"`) if used?